.github/workflows/test.yml

```diff
@@ -65,28 +65,28 @@ jobs:
       - name: Build
         run: GOOS=linux CGO_ENABLED=1 GOARCH=${{ matrix.goarch }} CC=${{ matrix.gcc }} LEDGER_ENABLED=false make build
 
-  # TODO: disable test-race. please enable this after fixing concurrent checkTx
-  # test-cosmovisor:
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@v2
-  #     - uses: actions/setup-go@v2.1.4
-  #       with:
-  #         go-version: 1.15
-  #     - name: Display go version
-  #       run: go version
-  #     - uses: technote-space/get-diff-action@v5.0.1
-  #       id: git_diff
-  #       with:
-  #         PREFIX_FILTER: |
-  #           cosmovisor
-  #         PATTERNS: |
-  #           **/**.go
-  #           go.mod
-  #           go.sum
-  #     - name: Run cosmovisor tests
-  #       run: cd cosmovisor; make
-  #       if: env.GIT_DIFF
+# TODO: disable test-cosmovisor; this test uses uploaded binary(cosmos-sdk)
```

Comment:
In order to enable this, research on `test-cosmovisor` is needed. We have to do this as a separate task.